zaza/utilities/openstack.py

```diff
@@ -1123,6 +1124,52 @@ def get_application_config_keys(application):
     return list(application_config.keys())
 
 
+def get_current_os_release_pair():
+    """Return OpenStack Release pair name.
```

Comment:
Can you make it possible to specify which app to use for the release query? I think keystone is  a sensible default but there may be occasions (like testing upgrades) when it would be useful to target a different application.

Reply:
That is now possible.